### ISSUE TITLE
Flatten examples directory

### ISSUE THREAD

**Issue by charleskawczynski** [CliMA/ClimaAtmos.jl#300]
I think the examples would be a bit easier to navigate if we flatten the directory structure a bit more.

**Comment by szy21** [comment]
Should we close this?

**Comment by charleskawczynski** [comment]
I think navigation is much easier, and if the number of issues seems too noisy then I'm fine with closing, but I think there's a few things we can still move:

We could probably move a few more things into src, like:
 - `examples/hybrid/staggered_nonhydrostatic_model.jl`
 - Some of, or all of `examples/hybrid/TurbulenceConvectionUtils.jl` after #1327 and #1311 are merged.
 - `examples/hybrid/callbacks.jl`
 - Some of `examples/hybrid/types.jl`, but might be nicer/clearer after #1311

probably not the highest of priorities, though.

**Comment by szy21** [comment]
Thanks! We can keep it open then, I’m just closing issues that are not relevant anymore.

**Comment by szy21** [comment]
Superseded by #1383
